### PR TITLE
feat: backend for events #55 and videos #84

### PR DESCRIPTION
**Pull request by tomaszandel**
- backend for events
- backend for videos
- -7 and +7 months for events , as we dont have empty case,
-10 last videos taken from youtube
- calls goes FE-> BE -> APIs

### CODE REVIEW

**Comment by Marcelinna** [comment]
Can we format title ? to remove unwanted characters (&amp)
image
image

**Comment by tomaszandel** [comment]
> Can we format title? to remove unwanted characters (&amp) image image

done ;)

**Review by Marcelinna — APPROVED** [review]

File: backend/app/routes/web/events.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+from app.config import EOSC_EVENTS_API
+
+router = APIRouter()
+
+
+@router.get("/{event_date}")
+async def get_events(event_date: str):
+    url = f"{EOSC_EVENTS_API}"
+    url += f"{event_date}"
```
Comment (michal-szostak) [inline backend/app/routes/web/events.py:16]: Wrapping str in `f""` is redundant

**Review by michal-szostak — APPROVED** [review]

File: backend/app/routes/web/events.py (hunk 1 above)
Comment (tomaszandel) [reply backend/app/routes/web/events.py:16]: true

**Pull request merged**